### PR TITLE
Add debug log on successful text extraction

### PR DESCRIPTION
**Pull request by navarone-feekery**
## Closes https://github.com/elastic/enterprise-search-team/issues/5453

During some testing it came up that we should have debug logs for successful text extraction.
This PR adds that log.

### CODE REVIEW

File: connectors/utils.py [hunk 1]
```diff
@@ -858,4 +858,5 @@ async def parse_extraction_resp(self, filename, response):
             )
             return ""
 
+        logger.debug(f"Text extraction successful for `{filename}'.")
```
Comment (wangch079) [inline connectors/utils.py:861]:
```suggestion
        logger.debug(f"Text extraction is successful for `{filename}'.")
```

File: tests/test_utils.py [hunk 2]
```diff
@@ -677,7 +677,7 @@ def test_check_configured(self, mock_config, expected_result):
             assert extraction_service._check_configured() is expected_result
 
     @pytest.mark.asyncio
-    async def test_extract_text(self, mock_responses):
+    async def test_extract_text(self, mock_responses, patch_logger):
```
Comment (wangch079) [inline tests/test_utils.py:680]: I don't feel this is worth testing

**Review by wangch079 — DISMISSED** [review]

File: tests/test_utils.py (hunk 2 above)
Comment (navarone-feekery) [reply tests/test_utils.py:680]: I normally wouldn't either, but for this project we had a strict requirement of filenams being trackable in logs. I just wanted to ensure that the expected filename appears in the debug log.

**Comment by navarone-feekery** [comment]
@wangch079 thanks for the review, I added your change request. Could you take a look again please?

File: tests/test_utils.py [hunk 3]
```diff
@@ -695,9 +695,10 @@ async def test_extract_text(self, mock_responses):
             await extraction_service._end_session()
 
             assert response == "I've been extracted!"
+            patch_logger.assert_present("Text extraction successful for `notreal.txt'.")
```
Comment (navarone-feekery) [inline tests/test_utils.py:698]:
```suggestion
            patch_logger.assert_present("Text extraction is successful for `notreal.txt'.")
```

File: tests/test_utils.py [hunk 4]
```diff
@@ -695,9 +695,10 @@ async def test_extract_text(self, mock_responses):
             await extraction_service._end_session()
 
             assert response == "I've been extracted!"
+            patch_logger.assert_present("Text extraction is successful for `notreal.txt'.")
```
Comment (wangch079) [inline tests/test_utils.py:698]:
```suggestion
            patch_logger.assert_present("Text extraction is successful for 'notreal.txt'.")
```

File: tests/test_utils.py [hunk 5]
```diff
@@ -719,6 +720,7 @@ async def test_extract_text_with_file_pointer(self, mock_responses):
             await extraction_service._end_session()
 
             assert response == "I've been extracted from a local file!"
+            patch_logger.assert_present("Text extraction successful for `notreal.txt'.")
```
Comment (wangch079) [inline tests/test_utils.py:723]:
```suggestion
            patch_logger.assert_present("Text extraction successful for 'notreal.txt'.")
```

**Review by wangch079 — APPROVED** [review]

**Pull request merged**